app/controllers/find_local_council_controller.rb

```diff
@@ -73,44 +72,40 @@ def local_council
     @_local_council ||= fetch_local_council_from_areas(mapit_response.location.areas)
   end
 
-  def set_artefact_headers
-    set_slimmer_artefact_headers(dummy_artefact_with_hardcoded_links)
-  end
-
-  def dummy_artefact_with_hardcoded_links
-    # NOTE: We use a hash as a dummy artefact (which stores a variety of
-    #       hardcoded data) for the moment, until we decide if we
-    #       can store this and other related data on a content item
-    hardcoded_format
-      .merge(hardcoded_related_links)
-  end
-
-  def hardcoded_format
-    # NOTE: This is required to set 'dimension2' for Google Analytics tracking
-    {
-      "format" => "find-local-council",
-    }
-  end
+  def setup_content_item_and_navigation_helpers
+    @content_item = content_store.content_item(BASE_PATH).to_hash
+    # Remove the organisations from the content item - this will prevent the
+    # govuk:analytics:organisations meta tag from being generated until there is
+    # a better way of doing this. This is so we don't add the tag to pages that
+    # didn't have it before, thereby swamping analytics.
+    if @content_item["links"]
+      @content_item["links"].delete("organisations")
+    end
 
-  def hardcoded_related_links
-    {
-      "related" => [
-        {
-          "id" => "#{Frontend.govuk_website_root}/api/understand-how-your-council-works.json",
-          "content_id" => "df61f873-f42f-4fb9-8e8e-17fa6a583270",
-          "web_url" => "#{Frontend.govuk_website_root}/understand-how-your-council-works",
-          "title" => "Understand how your council works",
-          "format" => "guide",
-          "owning_app" => "publisher",
-          "in_beta" => false,
-          "updated_at" => "2014-10-22T16:24:06+01:00",
-          "group" => "subsection"
-        },
-      ]
-    }
+    # Patch in a hardcoded document_type for consistency in Google Analytics
+    @content_item["document_type"] = "find-local-council"
+
+    # Patch in some hardcoded related links
+    @content_item["links"]["ordered_related_items"] = [
```

Comment:
<img width="1245" alt="screen shot 2016-10-27 at 17 19 31" src="https://cloud.githubusercontent.com/assets/233676/19775807/922c9f18-9c69-11e6-80d1-feb6672a9d77.png">

https://github.com/alphagov/frontend/pull/1032 got merged and is now setting the correct link. We can remove this hash now.